app/concepts/cinema_halls/use_cases/create.rb

```diff
@@ -0,0 +1,21 @@
+module CinemaHalls
+  module UseCases
+    class Create
+      attr_reader :repository
+
+      def initialize(repository: CinemaHalls::Repository.new)
+        @repository = repository
+      end
+
+      def call(params:)
+        seats = CinemaHalls::UseCases::CreateSeats.new(params: params).call
+        params[:seats] = seats
+        params.delete(:rows)
```

Comment:
Is this a place for Processor? I'm not sure I get the idea of this element of concepts.

Reply:
@wmaciejak that's a good question :D

Reply:
Yes @kodowicz and @Sonickers! :) It's a perfect place for processor :)